coderd/database/migrations/000025_allow_template_name_re_use.up.sql

```diff
@@ -0,0 +1,4 @@
+DROP INDEX idx_templates_name_lower;
+CREATE UNIQUE INDEX idx_templates_name_lower ON templates USING btree (lower(name)) WHERE deleted = false;
+
+ALTER TABLE ONLY templates DROP CONSTRAINT templates_organization_id_name_key;
```

Comment:
This should be OK since we already have a partial unique index:

```
CREATE UNIQUE INDEX templates_organization_id_name_idx ON templates USING btree (organization_id, name) WHERE (deleted = false);
```

Reply:
I still think these are werid